src/msw/window.cpp

```diff
@@ -1986,16 +1954,71 @@ wxWindowMSW::DoMoveSibling(WXHWND hwnd, int x, int y, int width, int height)
         }
     }
 
-    if ( !::MoveWindow((HWND)hwnd, x, y, width, height, IsShown()) )
+#if wxUSE_DEFERRED_SIZING
+    else if ( MSWIsPositionDirectlySupported(x, y) )
     {
-        wxLogLastError(wxT("MoveWindow"));
+        // if our parent had prepared a defer window handle for us, use it (unless
```

Comment:
The "unless ..." part of the comment doesn't seem to make sense any longer now that we don't test for `IsTopLevel()` here any more.